Documentation/ApiOverview/Logging/Writers/Index.rst

```diff
@@ -172,15 +172,17 @@ the :ref:`PhpErrorLog <logging-writers-php>` writer).
 Usage in Custom Class
 ---------------------
 
-All log writers can be used in your own classes. You can initialize the loggers in the __construct method of your class :code:`\MyDomain\MyExtension\MyFolder\MyClass`.
+All log writers can be used in your own classes. If the service is configured to use autowiring 
+you can initialize the loggers in the __construct method of your class :code:`\MyDomain\MyExtension\MyFolder\MyClass`) . 
```

Comment:
```suggestion
you can inject a logger into the :php:`__construct` method of your class :php:`\MyVendor\MyExtension\MyFolder\MyClass`) since TYPO3 v11 LTS. 
```

I would prefer "inject" instead of initialize. Also we use "MyVendor" as vendor example throughout the documentation.